twiml/voice/pay/prompt/full-example-ach/output/full-example-ach.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Response>
+	<Pay input="dtmf" timeout="5" maxAttempts="3" paymentMethod="ach-debit" language="en-US">
+		<Prompt for="bank-routing-number">
+			<Say>
+				Please enter your bank routing number.
+			</Say>
+		</Prompt>
+
+		<Prompt for="bank-routing-number" errorType= "timeout">
+			<Say>
+			I am sorry I didn't hear anything. Please enter your bank routing number.
```

Comment:
Probably another issue with the generator. This is the only case where the text is not indented.